File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 18 and 19 July 2020/01. Agency Profit/01. Agency Profit.py
company = input()
tickets, children_tickets = int(input()), int(input())
ticket_price, tax = float(input()), float(input())

adults = ticket_price + tax
children = ticket_price * 0.3 + tax
all_tickets_price = adults * tickets + children * children_tickets
print(f'The profit of your agency from {company} tickets is {all_tickets_price * 0.2:.2f} lv.')

'''
WizzAir
15
5
120
40

'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 18 and 19 July 2020/01. Agency Profit/01. Agency Profit.js
function agencyProfit(arg) {
  let [company, tickets, childrenTickets, ticketPrice, tax] = arg.map(el => Number(el) || el)
  let adults = ticketPrice + tax
  let children = ticketPrice * 0.3 + tax
  let allTicketsPrice = adults * tickets + children * childrenTickets
  return (`The profit of your agency from ${company} tickets is ${(allTicketsPrice * 0.2).toFixed(2)} lv.`)
}

console.log(agencyProfit(["WizzAir", "15", "5", "120", "40"]))

